src/test/groovy/org/gradle/profiler/mutations/ApplyChangeToAndroidLayoutFileMutatorTest.groovy

```diff
@@ -47,6 +50,7 @@ class ApplyChangeToAndroidLayoutFileMutatorTest extends AbstractMutatorTest {
         def mutator = new ApplyChangeToAndroidLayoutFileMutator(sourceFile)
 
         when:
+        mutator.beforeScenario(scenarioContext)
         mutator.afterScenario(scenarioContext)
```

Comment:
I think these tests don't make much sense now. We could make some random change between `beforeScenario()` and `afterScenario()` to make it meaningful.

But since this logic of "saving and restoring the original text" happens for every file change mutator, it's not really useful to test it so many times.

Reply:
Maybe a better option would be to have a method to create the mutator in every test that also calls `mutator.beforeScenario()` and `mutator.beforeBuild()`?

Reply:
I see your point., but there are actually tests with: ```reverts changes when nothing has been applied``` :D. So they already tested before that there is no action.

I left them for now, since they are there, do you think I should delete them to not continue with that pattern in future?